docs/user-guide/sdh/README.md

```diff
@@ -0,0 +1,79 @@
+
+* **Complex Release Management**: You need a centralized solution to manage the deployment of software updates across various environments and ensure smooth collaboration between different teams involved in the release process.
+
+* **Enhanced Visibility and Monitoring**: You need improved visibility into the release process for stakeholders such as Release Managers, Developers, DevOps teams, System Reliability Engineers (SREs), Tenant Points of Contact (POCs), and other stakeholders. They need to monitor releases, debug issues, and have access to current operational statuses.
+
+* **Standardized Deployment Processes**: You want to standardize and automate deployment processes to ensure error-free releases and streamline operations across multiple tenant environments.
+
+* **Comprehensive Release Documentation**: You require detailed documentation and checklists for each release, specifying versioning, dependencies, configuration changes, and deployment prerequisites to ensure consistent and successful deployments.
+
+---
+
+## Advantages
+
+Devtron's Software Distribution Hub goes beyond basic deployment by providing end-to-end release management. Deployments involving manual processes might be prone to human error. However, Software Distribution Hub automates the [rollout](#rollout) process by enforcing [requirements](#requirements) for each release, and not just for one environment but multiple tenant environments.
+
+### Normal Deployment vs SDH
+
+| Aspect                                     | Normal Deployment                            | Software Distribution Hub                                      |
+|--------------------------------------------|----------------------------------------------|----------------------------------------------------------------|
+| **Release Management**                     | No versioned deployments                     | Centralizes versioning and deployment into a unified platform  |
+| **Visibility**                             | Limited visibility (Apps/App Groups)         | Comprehensive visibility                                       |
+| **Automated and Standardized Deployments** | Relies on manual scripts or basic automation (jobs) | Automates and standardizes deployments for consistency  |
+| **End-to-End Release Management**          | Focuses on pushing changes quickly           | Manages entire release lifecycle from planning to post-release |
+
+---
+
+## Concepts
+
+Devtron's Software Distribution Hub has 2 sections:
+
+* [Tenants](./tenants.md)
+* [Release Hub](./release-hub.md)
+
+
+Feel free to familiarize yourself with the following concepts (terms) before you proceed to Software Distribution Hub.
+
+### Tenants
```

Comment:
Are we planning to add the new entities to glossary?

Reply:
No, @abhibhaw
These seem to be SDH-specific terms at the moment, so we intend to keep all these terms in this doc itself. In future, if we reuse any of these terms outside the SDH doc, we will put it in the glossary. Abhi ke liye, it is a one-stop vocabulary section for educating our users upfront.